tests/Framework/Bootloader/Http/HttpBootloaderTest.php

```diff
@@ -21,7 +24,17 @@ final class HttpBootloaderTest extends BaseTestCase
 {
     public function testHttpBinding(): void
     {
-        $this->assertContainerBoundAsSingleton(Http::class, Http::class);
+        $this->getContainer()->runScope(new Scope(ScopeName::Http), function (Container $container): void {
```

Comment:
Looks like you can use TestScope attribute here

Reply:
Yes, you're right